gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -161,6 +161,11 @@ secrets:
 # Gateway Kafka server
 #kafka:
 #  enabled: false
+
+#  # Host Mode
```

Comment:
I'm not adding any more documentation here? the mode will be described in a documentation I guess?